back/functions/filterAndRank.js

```diff
@@ -13,17 +9,34 @@ async function getMean(experienceID, feedbackID) {
     const sum = reviews.reduce((sum, score) => sum + score.toJSON().score, 0);
     return sum / reviews.length;
   }
-  return 0;
+  return null;
+}
+
+// Find max points an experience can get
+function getMaxPoints(numberPreferences) {
+  let result = 0;
+  for (let i = 1; i <= numberPreferences; i++) {
```

Comment:
Any particular reason we're starting at 1 instead of 0? :D 

Also it seems like this whole function could just `return numberPreferences.length`

Reply:
That works amazingly!!! Thanks for that!

Reply:
About starting at 0, I had no particular reason 😅 
But now that you mention it, it is probably better to have it that way